__tests__/lib/site-import.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import siteImport from 'lib/site-import';
+import dbFileImport from 'lib/site-import/db-file-import';
+
+describe( 'site import tests', () => {
+	describe( 'siteImport', () => {
+		it( 'should should have an object called dbSiteImport', () => {
```

Comment:
Duplicate should here
```suggestion
		it( 'should have an object called dbSiteImport', () => {
```